### DISCUSSION TITLE
Passwort anzeigen

### DISCUSSION THREAD

**Discussion by ElbPsi** [fabiankachlock/zwoo#258] [Ideas]
Hallo Team,
bei meinen komplizierten Passworten wäre es schön, wenn ich diese anzeigen lassen könnte, wenn ich diese eingebe.
Ihr kennt sicher das "Auge" bei einigen PW-Eingaben.

Grüße, Elb

**Comment by fabiankachlock** [comment]
Hallo Elb,
vielen Dank für die Anmerkung.

Wir werden eine Möglichkeit, das Passwort bei der Eingabe anzuzeigen, zeitnah umsetzen.

zwoo Team

**Reply by fabiankachlock** [reply]
linked: #265; ZWOO-87

**Comment by fabiankachlock** [comment]
Lösungsversion: v1.0.0-beta.5 (Release voraussichtlich am Ende der Woche)
